app/_locales/en/messages.json

```diff
@@ -860,6 +866,12 @@
   "curveMediumGasEstimate": {
     "message": "Market gas estimate graph"
   },
+  "custodian": {
+    "message": "Custodian"
+  },
+  "custodianAccount": {
+    "message": "Custodian Account"
```

Comment:
Is "Custodian Account" a page or noun? We use sentence case for most things with some few exceptions cc @coreyjanssen 

Content guidelines here https://www.notion.so/MetaMask-Content-Design-Style-Guide-7d0b7579786540afbe02780ebb203254?pvs=4#01ef60533da54d5f9860c0f65f487ac2

Reply:
+1, please make this sentence case :)